src/platform/user/profile/vet360/components/EmailField/VAPEmailField.jsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { API_ROUTES, FIELD_NAMES } from 'vet360/constants';
+
+import VAPProfileField from 'vet360/containers/VAPProfileField';
+
+import EmailEditView from './EmailEditView';
+import EmailView from './EmailView';
+
+const formSchema = {
```

Comment:
I'm not sure where the best place for this is but since it's only used by the child `VAPProfileField` that's configured in this component I'm okay with it here. But if you've got an idea you prefer I'm open to that.

Reply:
There's a lot about how all the VAP/Vet360 components deal with props that I'm not crazy about, but that's a different issue and is something I'd like to address a little later. But the fact that we define the schema here, pass it to the `VAPProfileField` so that that the `VAPProfileField` can just pass it to the `EmailEditView` so it can _then_ be passed to a `VAPEditView` feels like spaghetti.  I also don't like the pattern of blindly passing all props down to a child component. And that's done a lot with these components. But like I said on Slack, I want to get things working in Profile 2.0 before I start refactoring.